scss/_reboot.scss

```diff
@@ -10,37 +10,36 @@
 
 // Document
 //
-// Change from `box-sizing: content-box` so that `width` is not affected by `padding` or `border`.
+// Inherit box-sizing from parents.
+
 *,
 *::before,
 *::after {
-  box-sizing: border-box;
-}
-
-// Shim for IE11
```

Comment:
Do we want to keep this for IE11?

Reply:
This was just moved to the bottom, see:
https://github.com/twbs/bootstrap/blob/d81ab18f3f6f704f8146132e99cd298a0c9327f5/scss/_reboot.scss#L523-L531